File: src/main/java/io/vertigo/shiny/components/data/calendar/ShinyCalendar.java
package io.vertigo.shiny.components.data.calendar;

import io.vertigo.shiny.components.ShinyComponent;

public record ShinyCalendar(
		//private final List<LocalDate> highlightedDates;
		int year,
		int month) implements ShinyComponent {

	@Override
	public String type() {
		return "calendar";
	}
}

File: banshee/src/models/data/calendar/ShinyCalendar.ts
import { ShinyComponent } from '../../../ShinyComponent';

export class ShinyCalendar implements ShinyComponent {
  year: number;
  month: number;
  type: string = 'calendar';

  constructor(year: number, month: number) {
    this.year = year;
    this.month = month;
  }
}
